README.md

```diff
@@ -26,9 +26,10 @@ Please see the link below for more detail on TypeScript 3.8.x:
 * [TypeScript 3.8.x](https://www.npmjs.com/package/typescript)
 
 
-
 ## Getting Started
 
+Please see the [Quickstart guide for the Amazon QLDB Driver for Node.js](https://docs.aws.amazon.com/qldb/latest/developerguide/driver-quickstart-nodejs.html).
+
 To use the driver, in your package that wishes to use the driver, run the following:
```

Comment:
This section isn't in all the drivers, how are we going to unify this?

Reply:
We should add to the others that don't have it.